charts/grafana/templates/grafana-auth-sidecar-configmap.yaml

```diff
@@ -1,5 +1,5 @@
 ################################
```

Comment:
Nitpick, but while we're cleaning up these three comments, can we make the hash lines consistent?
```suggestion
#################################
```

Reply:
Works for me. They were consistently inconsistent everywhere so I left them 🤷‍♂️